terasoluna-gfw-functionaltest-env/src/main/resources/logback.xml

```diff
@@ -93,33 +93,32 @@
     <level value="info" />
   </logger>
 
-  <logger name="org.hibernate.engine.transaction">
-    <level value="debug" />
+  <logger name="org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping">
+    <level value="trace" />
   </logger>
 
-  <logger name="jdbc.sqltiming">
-    <level value="debug" />
+  <logger name="org.springframework.jdbc.core.JdbcTemplate">
+    <level value="trace" />
   </logger>
-
-  <logger name="org.springframework.jdbc.datasource">
+  <logger name="org.springframework.jdbc.datasource.DataSourceTransactionManager">
     <level value="debug" />
   </logger>
-  <logger name="org.springframework.orm.jpa">
-    <level value="debug" />
+
+  <logger name="java.sql.Connection">
+    <level value="trace" />
   </logger>
-  <logger name="org.springframework.transaction">
+
+  <logger name="java.sql.PreparedStatement">
```

Comment:
Why do you need `java.sql.Connection`, `java.sql.PreparedStatement` ?

Reply:
As a result of checking, it was deleted because it was not necessary.

f51ba523ab10d05c301a582d6ae06416b847a0bf